### DISCUSSION TITLE
View a single CSV file? I really like how WinMerge displays CSV's

### DISCUSSION THREAD

**Discussion by mejobloggs-cw** [WinMerge/winmerge#2316] [General]
I really like how WinMerge displays CSVs.

Can I use it just for general CSV viewing for a single file? Without having a comparison?

Would be so useful for me, instead of viewing CSV in Excel or Notepads

**Comment by sdottaka** [comment]
Unfortunately WinMerge does not have a single pane mode.

You can display the table comparison window by specifying two same CSV files or by specifying only one CSV file in Select Files or Folders window and pressing the compare button.
You can also expand one pane by dragging the pane borders.

**Comment by meteorquake** [comment]
Although slightly different from Winmerge's, it's worth noting that other programs such as Notepad++ and Notepad4 have interesting ways of presenting CSV, see eg screenshots at https://www.google.com/search?q=notepad+csv&udm=2
